frappe/erpnext
# fix: Paid invoices showing in AR/AP report

Paid Multicurrency Invoices still showing up in AR/AP report

Steps to test:
1. Make a multi-currency sales invoice from INR to USD at an exchange rate of 70
<img width="1306" alt="Screenshot 2021-12-02 at 6 40 30 PM" src="https://user-images.githubusercontent.com/42651287/144429817-b21792bb-c748-4055-a0cd-acecaa8c6643.png">

2. Make a standalone payment against the same customer at an exchange rate of 65
<img width="1302" alt="Screenshot 2021-12-02 at 6 44 04 PM" src="https://user-images.githubusercontent.com/42651287/144429977-ac89e0d3-e708-44af-b3bd-cd7bb86e6ce6.png">

3. Reconcile this payment against the Invoice using the payment reconciliation tool
<img width="1344" alt="Screenshot 2021-12-02 at 6 44 42 PM" src="https://user-images.githubusercontent.com/42651287/144430348-961597b0-360d-4d06-8d5a-8775cac4e85a.png">

The status Of the invoice will be changed to paid but it is still visible in the Accounts Payable Report if you'll check using the inline data table filters

<img width="1334" alt="Screenshot 2021-12-02 at 7 03 26 PM" src="https://user-images.githubusercontent.com/42651287/144433209-8805f179-987c-420e-b192-9c537f79ae2c.png">


Solution: Check for outstanding amount in both company and party currency as well
